### DISCUSSION TITLE
"pnpm i" exit with " ERROR  pattern.replace is not a function"

### DISCUSSION THREAD

**Discussion by johann-taberlet** [adonisjs/core#3780] [General]
Hey there,
because I'm still learning a lot of things, I'm not able to determine if it's a real bug or if it's something related to my config so I first post here before opening an issue.

What I did :
`pnpm create adonis-ts-app hello-world`

- api
- eslint true
- prettier true

```
cd hello-world
pnpm i 
```

And I get this instantly `ERROR  pattern.replace is not a function`

I know packages are already installed during the creation of the project but I was testing this command in a context of a pnpm monorepo by deleting package-lock.json and ./node_modules folder. But either by creating a "normal" project without deleting. anything, I have the same problem.

```
node -v
v16.15.1
pnpm -v
7.3.0
```
MacOS Monterey 12.4 with M1 Pro chip

Any ideas on whats wrong here ?

**Comment by johann-taberlet** [comment]
Ok my bad, it must be related with my pnpm workspaces thing because outside of it, it works as intended...

Sorry for that, you can delete it if you like.

**Reply by thetutlage** [reply]
No worries :)
